src/main/resources/swagger/cromiam.yaml

```diff
@@ -43,8 +59,10 @@ paths:
         '500':
           description: Internal Error
       security:
-        - google_oauth:
+        - googleoauth:
```

Comment:
total out of curiosity but why these removals of the `_`? required by the new version?